data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -486,4 +486,52 @@
+'EHRConsentPII_TXandMAHIPPA','TXandMAHIPPA_IDoAgree','TXandMAHIPPA_IDoNotAgree','EHRConsentPII_ILHIPPAWitnessSignature',
+'EHRConsentPII_TodaysDateILHIPPAWitness','EHRConsentPII_ILHIPPAReminder','ILHIPPAReminder_Yes','ILHIPPAReminder_No',
+'General Consent (Suppress The Following)','ExtraConsent_FitnessTrackerVideo','ConsentPII_ExtraConsent',
+'ExtraConsent_18YearsofAge','18YearsofAge_18YearsYes','18YearsofAge_18YearsNo','ExtraConsent_PhysicalMeausrementsVideo',
+'ExtraConsent_WelcomeVideo','ExtraConsent_HealthDataVideo','ExtraConsent_SamplesVideo','ExtraConsent_HealthRecordVideo',
+'ExtraConsent_Signature','ExtraConsent__CABoRTodaysDate','ExtraConsent_KeepinginTouchVideo',
+'ExtraConsent_OtherHealthDataVideo','ExtraConsent_CABoRSignature','ExtraConsent_DataSharingVideo',
+'ExtraConsent_EmailACopyToMe','ExtraConsent_DataShaingVideo','ExtraConsent_RisktoPrivacyVideo',
+'ExtraConsent_AllOfUsPurpose','AllOfUsPurpose_GiveMedicalAdvice','AllOfUsPurpose_HelpScientistsDiscover',
+'ExtraConsent_TakePartAllOfUs','TakePartAllOfUs_TakingPartIsVoluntary','TakePartAllOfUs_TakingPartIsRequired',
+'ExtraConsent_CanIWithdraw','CanIWithdraw_ICanWithdraw','CanIWithdraw_ICannotWithdraw',
+'ExtraConsent_PrivacyGuaranteeQuestion','PrivacyGuaranteeQuestion_PrivacyGuaranteed',
+'PrivacyGuaranteeQuestion_RisksAreMinimized','ExtraConsent_ReadyOrNeedHelp','ReadyOrNeedHelp_Ready',
+'ReadyOrNeedHelp_NeedHelp','ExtraConsent_AgreeToConsent','ConsentPII_SanDiegoSitePairing',
+'SanDiegoSitePairing_SanDiegoBloodBank','ConsentPII_SitePairing','SitePairing_CurrentlyReside',
+'CurrentlyReside_ResideAZSArizona','CurrentlyReside_ResideAZCArizona','CurrentlyReside_ResideNeither',
+'ResideAZSArizona_AZSArizonaSpecific','ResideAZCArizona_AZCArizonaSpecific','AZCArizonaSpecific_BannerBehavioralHealth',
+'AZCArizonaSpecific_AZBannerGoldfieldMedical','AZCArizonaSpecific_AZBannerCasaGrande',
+'AZCArizonaSpecific_AZBannerIronwoodMedical','AZSArizonaSpecific_AZUofArizona','AZSArizonaSpecific_AZBannerGatewayMed',
+'AZSArizonaSpecific_AZBannerThunderbirdMed','AZSArizonaSpecific_AZBannerMDAnderson',
+'AZSArizonaSpecific_AZBannerDelEWebbMed','AZSArizonaSpecific_AZBannerBoswellMed','AZSArizonaSpecific_AZUAHN',
+'AZSArizonaSpecific_AZ_no','AZSArizonaSpecific_AZUPH','AZSArizonaSpecific_AZUofACancerCenter',
+'AZSArizonaSpecific_AZBannerDesertMed','AZSArizonaSpecific_AZ_yes','AZSArizonaSpecific_AZBannerUMed',
+'AZSArizonaSpecific_AZBannerEstrellaMed','AZSArizonaSpecific_AZUMI','AZCArizonaSpecific_AZBannerBaywoodMed',
+'AZCArizonaSpecific_AZBannerHeartHospital','AZCArizonaSpecific_CardonChildrensMedical','ExtraConsent_21YearsofAge',
+'21YearsofAge_21YearsYes','21YearsofAge_21YearsNo','ExtraConsent_19YearsofAge','19YearsofAge_19YearsYes',
+'19YearsofAge_19YearsNo','ReceiveCare_PIIState','VeteransHealthAdministration_Consent','VeteransHealthAdministration',
+'VeteransHealthAdministration_CareYes','VeteransHealthAdministration_CareNo','StateOfResidence','ExtraConsent_DNAVideo')
```

Comment:
Is there a way to retrieve this full list (or most of it) from the Concept table with the right filters? I would prefer to reduce the number of hardcoded concepts

Reply:
@nishanthpp93 so the list that the RDR team gets is from [this google document](https://docs.google.com/document/d/1q8XqUZJUWbEr4IXXdvzFmPV84qs8uLgbfZ6ffNGWWys/edit). Now, how do we best handle this list? say if someone decided to add more concepts to it?

Reply:
@rfrancis1 That's a good question. I'll raise this in eng sync and get back to you.

Reply:
@nishanthpp93 Did you have some feedback on this?

Reply:
@rfrancis1 There is a spreadsheet that we can use as a table that @lrwb-aou mentioned. However I think we can create a separate one for this purpose internal to the curation team. I'll set it up and reach out to you.

Reply:
@rfrancis1 I've set it up. Can you replace the subquery to use the table `operational_data.operational_ehr_consent`? I've also set up `operational_general_consent` if needed. They use a separate spreadsheet I've set up in the curation team drive.